lib/extensions/ar_miq_set.rb

```diff
@@ -56,6 +56,9 @@ module ActsAsMiqSet
                               :scope => [:set_type, :userid, :group_id],
                               :if    => proc { |c| c.class.in_my_region.exists?(:name => c.name) }
     validates_presence_of     :description
+    validates_uniqueness_of   :description,
+                              :scope => :set_type,
+                              :if    => proc { |c| [MiqPolicySet, MiqAlertSet].include?(c.class) }
```

Comment:
@lfu Can this be moved to the classes that need it? This way the code in `ActsAsMiqSet` can remain generic.